com/truckmuncher/api/trucks.proto

```diff
@@ -77,6 +85,19 @@ message ActiveTrucksResponse {
          * Value in the range [-180, 180]
          */
         required double longitude = 3;
+        /** 
+         * Indicates whether the truck's location is verified or not.
+         */
+        optional bool verified = 4;
+        /**
+         * Indicates the current upvote/downvote score.
+         */
+        optional int score = 5;
+        /**
+         * Signifies the user's vote for the truck. Positive numbers are upvotes, negatives are downvotes, anything else 
+         * (null or 0) means that the user has not provided a vote.
+         */
+        optional bool userVote = 6;
```

Comment:
The type and the documentation for this don't line up. We also have to make sure that it's super clear that null is a valid value here and that client's shouldn't blindly apply the default (which gets auto-generated)